optional_plugins/robot/avocado_robot/__init__.py

```diff
@@ -17,8 +17,6 @@
 import re
 import subprocess
 
-from StringIO import StringIO
-
```

Comment:
Nice bonus commit, but when you are on the cleanup, you should fix all or none of the pylint issues 😉

```
C0111:   1,0: Missing module docstring
R0913:  40,4: Too many arguments (6/5)
W0613:  45,17: Unused argument 'external_runner'
W0702:  97,8: No exception type(s) specified
W0621: 112,12: Redefining name 'test' from outer scope (line 25)
...
Global evaluation
-----------------
Your code has been rated at 8.97/10 (previous run: 8.97/10, +0.00)
```

What do you say?